chapters/en/slides/module7/module7_03.md

```diff
@@ -0,0 +1,389 @@
+```
+
+<iframe src="/module7/charts/03/unnamed-chunk-6.html" width="100%" height="420px" style="border:none;">
+</iframe>
+
+Notes: So far we have seen how to select categorical values from
+dropdown menus and radio buttons. To instead highlight based on
+quantitative values, we can use a slider.
+
+Adding a slider is similar to adding a dropdown or set of radio buttons,
+and we do it via `binding_range()`. Here we link the slider to the same
+column as on the x-axis, so that you can see how it works.
+
+If you drag the slider around, you can see that the default behavior is
+to only highlight points that are of the exact values as the slider.
+
+This is useful for selection widgets like the dropdown, but for the
+slider we want to make comparisons such as bigger and smaller than.
+
+Let’s see how to do that in the next slide.
+
+---
+
+## Highlighting points smaller or bigger than a slider value
+
+``` python
+points.encode(
+    opacity=alt.condition(
+        alt.datum.Rotten_Tomatoes_Rating < select_rating.Rotten_Tomatoes_Rating,
+        alt.value(0.7), alt.value(0.1))
+).add_selection(select_rating)
+```
+
+<iframe src="/module7/charts/03/unnamed-chunk-7.html" width="100%" height="420px" style="border:none;">
+</iframe>
+
+Notes: To highlight values that are smaller than the slider, we need to
+compare the current value of the slider with the value of the data
+points.
+
+The current value of the slider can be accessed via
+`select_rating.Rotten_Tomatoes_Rating`, and Altair has a special `datum`
+object to access values from different columns in the dataframe. To
+compare against the column with the Rotten Tomatoes Ratings, we use
+`alt.datum.Rotten_Tomatoes_Rating`.
+
+This might seem cumbersome at first, but it is necessary to avoid
+ambiguity with only writing `"Rotten_Tomatoes_Rating"`, which in
+comparison expressions means just the string and is not a reference to a
+column in the dataframe.
+
+Now that we drag the slider you can see that all the points are
+highlighted that have a value less than the slider value.
+
+---
+
+## Customizing a slider widget
+
+``` python
+slider = alt.binding_range(name='Tomatometer ', min=10, max=60, step=10)
+select_rating = alt.selection_single(
+    fields=['Rotten_Tomatoes_Rating'],
+    bind=slider,
+    init={'Rotten_Tomatoes_Rating': 20})
+
+points.encode(
+    opacity=alt.condition(
+        alt.datum.Rotten_Tomatoes_Rating < select_rating.Rotten_Tomatoes_Rating,
+        alt.value(0.7), alt.value(0.1))
+).add_selection(select_rating)
+```
+
+<iframe src="/module7/charts/03/unnamed-chunk-8.html" width="100%" height="420px" style="border:none;">
+</iframe>
+
+Notes: Just as with dropdowns we can set an explicit start value to have
+an active selection when the chart first appears.
+
+For slider widgets we can also define the range and step size for the
+slider as we have done in this slide.
+
+Now that we drag the slider handle it jumps ten steps at a time, instead
+of the default single step.
```

Comment:
The last sentence begs the question, what is the default step size?

```suggestion
Notes: Just as with dropdowns we can set an explicit start value to have
an active selection when the chart first appears. Here we have set the 
chart so that the slider will be set at 20 when the chart first loads via
setting `init={'Rotten_Tomatoes_Rating': 20})` in `alt.selection_single`.

For slider widgets we can also define the range and step size for the
slider as we have done here, setting the minimum to be 10, the maximum
to be 60, and a step size of 10.

Now that we drag the slider handle it jumps ten steps at a time, instead
of the default single step.
```